tests/integration/utils/fixtures.py

```diff
@@ -128,6 +128,12 @@ def build_basic_market(
     )
     vega.wait_for_total_catchup()
 
+    vega.update_network_parameter(
+        proposal_wallet=MM_WALLET.name,
+        parameter="transfer.fee.factor",
+        new_value="0",
```

Comment:
If this is non-zero normally we can probably just change it in `genesis.json`, seems like we probably want it as 0 by default in market sim